src/test/java/core/basesyntax/model/ShopTransactionTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ShopTransactionTest {
+    private ShopTransaction shopTransaction;
```

Comment:
same here, remove ShopTransactionTest

we need to write test to check som logic

Reply:
Ok